pkg/harvester/dialog/ConfirmExecutionDialog.vue

```diff
@@ -0,0 +1,194 @@
+        }
+
+        return res;
+      }, '');
+    },
+
+    nameToMatch() {
+      return this.resources[0].nameDisplay;
+    },
+
+    plusMore() {
+      const remaining = this.resources.length - this.names.length;
+
+      return this.t('dialog.confirmExecution.andOthers', { count: remaining });
+    },
+
+    type() {
+      const types = new Set(this.resources.reduce((array, each) => {
+        array.push(each.type);
+
+        return array;
+      }, []));
+
+      if (types.size > 1) {
+        return this.t('generic.resource', { count: this.resources.length });
+      }
+
+      const schema = this.resources[0]?.schema;
+
+      if ( !schema ) {
+        return `resource${ this.resources.length === 1 ? '' : 's' }`;
+      }
+
+      return this.$store.getters['type-map/labelFor'](schema, this.resources.length);
+    },
+
+    applyDisabled() {
+      return this.confirmName !== this.nameToMatch;
+    },
+
+    protip() {
+      return this.t('dialog.confirmExecution.protip', { alternateLabel });
+    },
+  },
+
+  methods: {
+    escapeHtml,
+
+    close() {
+      this.confirmName = '';
+      this.errors = [];
+      this.$emit('close');
+    },
+
+    async apply(buttonDone) {
+      try {
+        for (const resource of this.resources) {
+          await resource.doActionGrowl(this.modalData.action, {});
+        }
+        buttonDone(true);
+        this.close();
+      } catch (e) {
+        this.errors = exceptionToErrorsArray(e);
+        buttonDone(false);
+      }
+    }
+  }
+};
+</script>
+
+<template>
+  <Card
+    class="prompt-related"
+    :show-highlight-border="false"
+  >
+    <template #title>
+      <h4 class="text-default-text">
+        {{ t('dialog.confirmExecution.title') }}
+      </h4>
+    </template>
+
+    <template #body>
+      <div class="pl-10 pr-10">
+        <span
+          v-clean-html="t(warningMessageKey, { type, names: resourceNames }, true)"
+        ></span>
+
+        <div class="mt-10 mb-10">
+          <span
+            v-clean-html="t('dialog.confirmExecution.confirmName', { nameToMatch: escapeHtml(nameToMatch) }, true)"
+          ></span>
+        </div>
+        <div class="mb-10">
+          <CopyToClipboardText :text="nameToMatch" />
+        </div>
+        <input
+          id="confirm"
+          v-model="confirmName"
+          type="text"
+        />
+        <div class="text-info mt-20">
+          {{ protip }}
+        </div>
+        <Banner
+          v-for="(error, i) in errors"
+          :key="i"
+        />
+      </div>
+    </template>
+
+    <template #actions>
+      <button
+        class="btn role-secondary mr-10"
+        @click="close"
+      >
+        {{ t('generic.cancel') }}
+      </button>
+      <AsyncButton
+        mode="apply"
+        class="btn bg-error ml-10"
+        :disabled="applyDisabled"
+        @click="apply"
+      />
+    </template>
+  </Card>
+</template>
+
+<style lang='scss' scoped>
+  .actions {
+    text-align: right;
+  }
+</style>
```

Comment:
Copy the dialog content from `https://github.com/harvester/harvester-ui-extension/blob/main/pkg/harvester/dialog/ConfirmRelatedToRemoveDialog.vue`.
We can modify the component if we prefer a simpler confirmation pop-up.